package.json

```diff
@@ -105,6 +105,7 @@
     "a11y:validate": "./scripts/validateAccessibilitySpecs.js",
     "build": "cd packages/gestalt-design-tokens && yarn build && cd ../gestalt && yarn build && cd ../gestalt-datepicker && yarn build && cd ../eslint-plugin-gestalt && yarn build && cd ../../docs && yarn build",
     "codemod": "jscodeshift --ignore-pattern=**/node_modules/**",
+    "run:codemod": "./scripts/codemod/entry.sh",
```

Comment:
@AlbertCarreras could we merge `run:codemod` and `codemod`?

Reply:
yep, good catch. DONE